group/assignment5/main.cpp

```diff
@@ -0,0 +1,194 @@
+#include <iostream>
+#include <cmath>
+#include <vector>
+#include <unordered_map>
+#include "codeu_test_lib.h"
+
+using namespace std;
+
+vector<char> findAlphabetCharacters(const vector<string> &dictionary) {
+    vector<bool> charsInDict((unsigned int)pow(2, CHAR_BIT), false);
+    vector<char> alphabetChars;
+    for (auto word : dictionary) {
+        for (char c : word) {
+            if (!charsInDict[c]) {
+                charsInDict[c] = true;
+                alphabetChars.push_back(c);
+            }
+        }
+    }
+    return alphabetChars;
+}
+
+bool createCharPriorityGraph(vector<string> &dictionary, unordered_map<char, vector<bool>> &charPriorityGraph, unordered_map<char, int> &numOfPriorCharacters) {
+    while (dictionary.size() > 0 && dictionary[0].size() == 0) {
+        dictionary.erase(dictionary.begin());
```

Comment:
Here a dictionary provided by user is changed, I am not sure, but it may not be always acceptable.